src/Microsoft.AspNetCore.Razor.Language/Intermediate/CSharpExpressionAttributeValueIRNode.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Razor.Language.Intermediate
+{
+    public sealed class CSharpExpressionAttributeValueIRNode : RazorIRNode
+    {
+        public override ItemCollection Annotations => ReadOnlyItemCollection.Empty;
```

Comment:
We didn't do that for `HtmlAttributeValueIRNode` so I was just keeping things consistent. Do we want to do this on other nodes as well?